### PR TITLE
zh-trans /reference/setup-tools/kubeadm/generated/kubeadm_config_imag…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images.md [hunk 1]
```diff
@@ -40,7 +29,12 @@ kubeadm config images [flags]
       <td colspan="2">-h, --help</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">镜像的帮助信息</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      help for images
+      -->
+      镜像的帮助信息
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images.md]:
```suggestion
      image 的帮助命令
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images.md [hunk 2]
```diff
@@ -69,22 +54,50 @@ kubeadm config images [flags]
...
     <tr>
       <td colspan="2">--rootfs string</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">[实验] 到'真实'主机根文件系统的路径。</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      [EXPERIMENTAL] The path to the 'real' host root filesystem.
+      -->
+      [实验性功能] 到 '真实' 主机根文件系统的路径。
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images.md]:
```suggestion
      [实验] 到 '真实' 主机根文件系统的路径。
```

**Comment by tanjunchen** [comment]
@xichengliudui  已经修改

**Pull request merged**